tagstudio/src/qt/modals/delete_unlinked.py

```diff
@@ -28,10 +28,10 @@
 class DeleteUnlinkedEntriesModal(QWidget):
     done = Signal()
 
-    def __init__(self, library: "Library", driver: "QtDriver"):
+    def __init__(self, driver: "QtDriver", tracker: MissingRegistry):
```

Comment:
though, this and possible the rest of the places, you would need to import the class for this type. 

```suggestion
    def __init__(self, driver: QtDriver, tracker: MissingRegistry):
```